### ISSUE TITLE
Quest wit and re-organization.

### ISSUE THREAD

**Issue by Notawallplan** [NillerMedDild/Enigmatica2Expert#79]
Requirements and links won't be changed (much), just position, titles, and descriptions.

**Comment by NillerMedDild** [comment]
I agree, the titles are boring. I like most of the descriptions though.
I'd definitely be willing to merge enhancements to titles, and possibly positions aswell.

I will however not make these changes myself in the foreseeable future.

**Comment by Barhandar** [comment]
It would be pretty nice to have actual quest descriptions for many quests in place of "lol use the mod's own book", since that, uh, invalidates the whole point of using quests to guide player's progression. 
As well as some of the books being frankly terrible and some questlines out of order (Astral Sorcery's quest for crystal being positioned after quests that require it, IIRC, or Blood Magic having slate quests behind orb quests despite orbs being technically optional).

**Comment by NillerMedDild** [comment]
Are there any quests in particular you think need better (actual) descriptions? For the most part, I expect the user to be able to figure things out with the mod's manual, and only describe what I have to, but I that could be improved upon.

**Comment by Barhandar** [comment]
For Blood Magic, besides the general rearrangement (separating altar-slate pairs and orbs, you technically only need t1 orb and t1 altar to advance to t5) it would be good to have description for Arcane Ashes mentioning that the circle also can be used for persistent effects that don't show up in JEI; mention Sentient Sword in Petty Gem/Hellfire Forge quests since it's leagues better for collecting demonic will than snares (and also mention that when you have a tartaric gem in inventory, picked up Demonic Will will be added to the gem's storage and wasted if that's full, and when you don't it will be put in inventory as item); and mention other sigils in Divination quest. Additionally mention maximum size of the altar structure in the altar quest (23x9x23 with the altar block being at 6th block from the bottom, in the center) . Also a quest for advanced runes (like Rune of Capacity needed for Astral Sorcery) would be good, mentioning that any basic rune in the Altar can be replaced with advanced one.
>For the most part, I expect the user to be able to figure things out with the mod's manual

I have the opposite expectation, in that a player should be able to go through the mod on quests and JEI alone, and only need the book for details or if they want to delve into the mod proper. Current questline _almost_ accomplishes that.
All because holy sky the Sanguine Scientem is _dense_ and in the bad way. I've seen exactly one guidebook worse than it and that was in an extremely non-mainstream mod and had a condensed version. Its "Demon Will and Snares" chapter is 12 pages of _fluff and more fluff_ when the crunch is "Demonic Will is a resource that can be contained in Tartaric Gems and obtained by killing mobs affected by Snares, or using Sentient Sword".

For Astral Sorcery make Ritual Altar require Attunement Altar quest (it's useless without attuned crystals) and mention that constellation effects are found in the book. Otherwise I'd need to replay it to point errors out.

For Thaumcraft, note in general or Essentia Smeltery quests that these produce Flux, cannot be prevented from producing Flux, and excess Flux in the chunk's aura spawns block-replacing (and thus potentially machine-destroying) Flux Rifts that then place Taint Seeds and cover the area in taint, and there is currently _no_ way to deal with these in Survival (Flux Sponge does it in Creative), as such Thaumcraft should be done somewhere you won't regret losing.

For Immersive Engineering, Projector quest shouldn't be gated by all the part block quests since it doesn't require them to craft and can still be useful for displaying some structures. Also, powertier quest descriptions are wrong: they're currently configured to transfer 1024/4096/16384 RF _per connection_ and 2048/8192/32768 RF _per wire network total_, not per connection like the book states.

**Comment by NillerMedDild** [comment]
All good points, I'll get the quests improved, thank you :)

**Comment by NillerMedDild** [comment]
Due to the amount of things I would have to cut out manually from your proposed changes, I won't be implementing any of them, it would simply take too long to splice out everything but the name changes.
